google_guest_agent/agentcrypto/crypto_util.go

```diff
@@ -38,6 +39,21 @@ func parseCertificate(cert []byte) (*x509.Certificate, error) {
 	return x509Cert, nil
 }
 
+// parsePvtKey validates the key is in valid format and returns the EC Private Key.
+func parsePvtKey(pemKey []byte) (*ecdsa.PrivateKey, error) {
+	key, _ := pem.Decode(pemKey)
```

Comment:
any trick in the `Decode()` error return here?

Reply:
`Decode()` does not return error and we don't expect/care any remaining bytes so its ignored.